python/cugraph-pyg/cugraph_pyg/data/cugraph_store.py

```diff
@@ -19,33 +19,27 @@
 from itertools import chain
 from functools import cached_property
 
-import warnings
 
 import numpy as np
+import pandas
 import cudf
+import cugraph
 
 from cugraph.utilities.utils import import_optional, MissingModule
 
-# FIXME remove these imports and replace PG with FeatureStore
-from cugraph.experimental import MGPropertyGraph
-
 # FIXME drop cupy support and make torch the only backend (#2995)
-cupy = import_optional("cupy")
+import cupy
```

Comment:
We should be safely getting this transitively, but we should really add `cupy` to the conda recipe now.

Reply:
I updated the recipe and removed this comment.